test/testData/mockReportData.json

```diff
@@ -165,7 +165,7 @@
     ],
     "kbMatches": [
         {
-            "category": "pharmacogenomic",
+            "category": "cancer predisposition",
```

Comment:
did we need another for pharmacogenomic? looks like this just replaces pharmacogenomic, but the change on kbMatches looks like `cancer predisposition` is a new field?

Reply:
This is just mock data used for our report upload test.
'pharmacogenomic' is still a valid category.
I just swapped it to 'cancer predisposition' in this test data to show that uploading a report with the category set to 'cancer predisposition' works.